driver/config/config.go

```diff
@@ -578,6 +579,18 @@ func (p *Config) IsInsecureDevMode() bool {
 	return p.Source().Bool("dev")
 }
 
+func (p *Config) IsBackgroundCourierEnabled() bool {
+	return p.Source().Bool("watch-courier")
+}
+
+func (p *Config) CourierExposeMetricsPort() int {
+	return p.Source().Int("courier.watch.expose-metrics-port")
```

Comment:
I don't think this is correct, shouldn't it be just `expose-metrics-port`?

Reply:
Yes, you're right. Forgot to update this bit.